README.md

```diff
@@ -1,14 +1,14 @@
 # Google Drive support for Jupyter Notebook
 
-This repository contains a custom
+This repository contains customs
```

Comment:
It contains 2 classes, Client Side and mixed content so should it be plurals without `a` ?